src/thing/thing.test.ts

```diff
@@ -1082,10 +1086,16 @@ describe("removeThing", () => {
       thingQuad,
       otherQuad,
     ]);
-    expect(
-      (datasetWithMultipleThings as SolidDataset & WithChangeLog)
-        .internal_changeLog
-    ).toBeUndefined();
+    // TODO: PMcB55 - removing this due to TS complaining (could be a mismatch
+    //  in type between `@types/rdf-dataset-indexed` and `rdf-js`...?
+    //   TS2352: Conversion of type 'DatasetIndexed<Quad, Quad>' to type
+    //   'WithChangeLog' may be a mistake because neither type sufficiently
+    //   overlaps with the other. If this was intentional, convert the
+    //   expression to 'unknown' first. Property 'internal_changeLog' is missing in type 'DatasetIndexed<Quad, Quad>' but required in type '{ internal_changeLog: { additions: Quad[]; deletions: Quad[]; }; }'.
+    // expect(
+    //   (datasetWithMultipleThings as SolidDataset & WithChangeLog)
+    //     .internal_changeLog
+    // ).toBeUndefined();
```

Comment:
Looks like `-indexed` doesn't (properly?) implement the RDF/JS interfaces. Something to solve in its implementation, but not relevant to this PR I don't think.

```suggestion
    expect(
      (datasetWithMultipleThings as SolidDataset & WithChangeLog)
        .internal_changeLog
    ).toBeUndefined();
```

Reply:
Done - 8f702c67